templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -319,6 +319,16 @@
             <td {$valueStyle}>{$form.payment_instrument_id.html} {help id="payment_instrument_id"}</td>
             </td>
           </tr>
+          {if $form.card_type}
```

Comment:
This is visible on offline contribution form when payment is not made through payment processor.